packages/react/src/button/Button.tsx

```diff
@@ -70,10 +71,27 @@ export const Button = forwardRef<HTMLButtonElement, ButtonProps>(
         </Text>
       );
 
-    const presetProps = presets[preset];
+    const presetProps = appearances[appearance];
     const finalColorScheme = colorScheme ?? presetProps.colorScheme;
     const finalVariant = variant ?? presetProps.variant;
 
+    const content = (
+      <Box
```

Comment:
why do we need this extra box? can't we merge the styles of `wrapper` and `button` ?

Reply:
addressed bhai